pages/upgrade/graylog-4.1.rst

```diff
@@ -0,0 +1,18 @@
+**************************
+Upgrading to Graylog 4.1.x
+**************************
+
+.. _upgrade-from-40-to-41:
+
+.. contents:: Overview
+   :depth: 3
+   :backlinks: top
+
+
+Breaking Changes
+================
+
+Changes to the Elasticsearch Support
+------------------------------------
+
+When you have version-probing for the used Elasticsearch version enabled, and Graylog starts up but can not connect to ES, the startup stopped immediately with v4.0 and prior. Starting from 4.1 the default behaviour is, that Graylog retries connecting with a delay until it can connect to Elasticsearch. See the Elasticsearch configuration page for details.
```

Comment:
This is slightly inaccurate. Before v4.0, there was no version probing, but prior to v4.0 _and_ in v4.0 _without_ version probing, Graylog continues connecting to ES until it is successfull, so this change is a breaking change compared to v4.0, but makes the behavior consistent with versions v3.3 and before _and_ with the behavior when version probing is disabled.